### ISSUE TITLE
Does this repo achieve SOTA accuracy after training for 300 epochs?

### ISSUE THREAD

**Issue by Whiax** [narumiruna/efficientnet-pytorch#32]
Hi
I would like to know if this code really achieves ~76% top1 accuracy with B0 on Imagenet after 300 epochs.
There are multiple things lacking (smooth cross entropy, augmentations, etc.) and I don't know how it can affect the results.
Thanks for you answer

**Comment by narumiruna** [comment]
No, this code doesn't achieve ~76% top1 accuracy with B0 on ImageNet.
